AntWars/GamePanel.cs

```diff
@@ -24,6 +24,15 @@ public GamePanel()
 
         public void start(Config.Configuration config)
         {
+            // set playernames in statistic
```

Comment:
Da das ja auch doppelt ist (in der view methode) sollte das auch eine eigene methode haben, das nicht soviel doppelter code da ist.